app/models/name/synonymy.rb

```diff
@@ -198,9 +198,9 @@ def more_popular(name)
 
   # (if no namings, returns created_at)
   def time_of_last_naming
-    @time_of_last_naming ||= begin
-      Naming.where(name_id: id).maximum(:created_at) || created_at
-    end
+    @time_of_last_naming ||= \
+      Naming.where(name_id: id).maximum(:created_at) ||
+      created_at
```

Comment:
Redundant begin/end deleted.
(It was redundant because `||` has precedence over assignment such as `||=`.)